roles/oneview_os_deployment_plan_facts/README.md

```diff
@@ -0,0 +1,32 @@
+## oneview_os_deployment_plan_facts
+Retrieve the facts about one or more of the OneView OS Deployment Plan Facts
```

Comment:
Remove Facts at the end

Reply:
this Read Me is about oneview_os_deployment_plan_facts hence kept it.